src/phasorpy/_phasorpy.pyx

```diff
@@ -795,7 +795,11 @@ cdef (float_t, float_t) _polar_from_reference(
 ) noexcept nogil:
     """Return polar coordinates for calibration from reference coordinates."""
     if fabs(measured_modulation) == 0.0:
-        return known_phase - measured_phase, <float_t> INFINITY
+        # return known_phase - measured_phase, <float_t> INFINITY
```

Comment:
Was it intended to keep previous return commented for comparison or other reason?

Reply:
It's a reminder that I'm not quite sure about the consequences of the change. It should now be closer to the way numpy handles division by zero.